packages/openactive-integration-tests/test/helpers/flow-stages/flow-stage-utils.js

```diff
@@ -136,25 +137,39 @@ const FlowStageUtils = {
    * @param {object} checks
    * @param {boolean} checks.doCheckSuccess If true, success checks will be run
    * @param {boolean} checks.doCheckIsValid If true, validation will be run
-   * @param {UnknownFlowStageType} flowStage
+   * @param {UnknownFlowStageType | BookRecipe} flowStageOrBookRecipe If this is a BookRecipe,
+   *   all stages within will be checked for validity/success.
```

Comment:
I was going to make new functions for running the BookRecipe but, surprisingly, it turned out easier to just make this function accept either a normal flow stage or the BookRecipe. Lemme know if this is simple or not